src/lib/Theme.ts

```diff
@@ -403,7 +406,7 @@ export default class Theme {
             const imageCdnUrl = await Theme.getImageCdnBaseUrl();
             const assetCdnUrl = await Theme.getAssetCdnBaseUrl();
             Theme.createVueConfig();
-            const assetHash = shortid.generate();
+            const assetHash = nanoid();
```

Comment:
What is the default hash length size ? Could see we can pass custom length to this function.

Reply:
yes we can pass, i have passed 9 size as currently size 9 is being used in every themebundle